docker-compose.yml

```diff
@@ -18,36 +18,9 @@ mesosmaster:
   links:
     - zookeeper
 
-slave1:
+slave:
   build: .
   privileged: true
-  ports:
-    - "5051:5051"
```

Comment:
Yes, sandbox links are all still working as expected without this. Can you think of any other reasons this port mapping exists? I'm not familiar with why its here. But you can't scale a service  in docker-compose if you have a port mapping set, which is why I removed it